src/components/Modal/Modal.jsx

```diff
@@ -152,6 +168,12 @@ Modal.propTypes = {
   wide: PropTypes.bool,
   /** this element will regain focus on modal close */
   previousFocus: PropTypes.node,
+  /** set a custom modal width */
+  customWidth: PropTypes.string,
+  /** remove the background so only the content shows */
+  noBackground: PropTypes.bool,
+  /** adds a close icon, the function to close remains in your app */
+  closeCross: PropTypes.func,
```

Comment:
Hey @gmzjuliana, do you think we can pass the following object instead of a function
```
{
  callback: PropTypes.func, 
}
```
and use `handleAction` with `dismissable`?

This is so we are following the same pattern as the other actions, and also leveraging the focus reset functionalities ;).

Reply:
Do you think we can call this `closeButton` instead? It feels a bit more generic.